Sources/TuistDependencies/SwiftPackageManager/Utils/PackageInfoMapper.swift

```diff
@@ -851,12 +866,23 @@ extension PackageInfo {
 }
 
 extension PackageInfo.Target {
-    var relativePath: RelativePath {
-        RelativePath(path ?? "Sources/\(name)")
+    func mainPath(packageFolder: AbsolutePath) throws -> AbsolutePath {
```

Comment:
maybe `targetSourcesPath`?

Reply:
It's already defined on `Target` so I wouldn't add target in the name.
And it's not the sources path, because you can still specify a different source in `Target.sources`, which would be relative from here

Reply:
`defaultTargetSourcesPath`? I still think `mainPath` is not descriptive enough but I agree it's hard to come up with something reasonable.